src/Sylius/Bundle/WebBundle/Resources/views/Frontend/Cart/summary.html.twig

```diff
@@ -43,10 +42,11 @@
         {% endfor %}
     </tbody>
     <tfoot>
-        {% if cart.adjustmentsTotal(promotionAdjustment) %}
+        {% set promotionTotal = cart.getPromotionsTotalRecursively() %}
```

Comment:
That is correct, but I mean [here] we don't display value of all promotion but only Order level.